sushi-config.yaml

```
@@ -22,6 +22,8 @@ dependencies:
     ans.fr.nos: 1.4.0
     hl7.fhir.fr.core: 2.1.0
     hl7.fhir.extensions.r5: 4.0.1
+    ans.fhir.fr.annuaire: 1.1.0-snapshot-6
```

```suggestion
    ans.fhir.fr.annuaire: 1.1.0
```

L'annuaire a été publié! :)